docs/pages/setup.md

```diff
@@ -49,8 +49,62 @@ Uju is a version control server designed specifically to work with the Lepsta Pl
 
 === "Windows"
 
-    !!! warning "Windows version  is not available yet"
+    !!! warning "We are working on the Windows version."
         We are still working on the Windows version and will make an announcement as soon as it is available.
+        However to get it working on windows you must activate the Windows Subsystem for Linux.
+
+    You must be on Windows 10 to activate Windows Subsystem for Linux for Uju to work.
+    
+    
+    Firstly, enable Windows Subsystem for Linux (WSL) using Settings. 
+
+    - Open Settings.
+    - Click on Apps.
+    - Under the "Related settings" area, click the Programs and Features option.
+
+    ![Click the Programs and Features option](/assets/images/features.PNG)
+
+    - Click the Turn Windows features on or off option from the left pane.
+
+    ![Turn Windows features on or off option from the left panel](/assets/images/windowsFeatures.PNG)
+    
+    - Check the Windows Subsystem for Linux option.
+    ![Check the Windows Subsystem for Linux option](/assets/images/windowsFeature.PNG)
+
+    - Click the OK button.
+    - Click the Restart now button.
+    Once you complete the steps, the environment will be configured to download and run the distros of Linux on Windows 10.
+
+    Head on to <a href="https://docs.microsoft.com/en-us/windows/wsl/install-win10https://docs.microsoft.com/en-us/windows/wsl/install-win10" target="_blank"> Microsoft Docs </a> to complete the remaining steps or to <a href="https://www.windowscentral.com/install-windows-subsystem-linux-windows-10" target="_blank"> Windows central </a> for more detailed instructions. 
+
+    !!! Note "Note"
+        If you are running Windows 10 version 1903 or 1909, open "Settings" from your Windows menu, navigate to "Update & Security" and select "Check for Updates". Your Build number must be 18362.1049+ or 18363.1049+, with the minor build # over .1049. Read more: WSL 2 Support is coming to Windows 10 Versions 1903 and 1909.
+        
+        See the troubleshooting instructions on  <a href="https://docs.microsoft.com/en-us/windows/wsl/install-win10https://docs.microsoft.com/en-us/windows/wsl/install-win10" target="_blank"> Microsoft Docs. </a>
+    
+    
+    Once you are done Installing Linux distros, Launch the linux distros that you installed eg. Ubuntu or Mint.
+    
+    ![Launch the Linux distros](/assets/images/launch.png)
+
+    - Create a username for the Linux distro and press Enter.
+    - Specify a password for the distro and press Enter.
+
+    ![Create a username and password](/assets/images/linux.PNG)
+
+
+    You can install Uju by running one of these commands on your terminal.
+        ```
+        curl https://lepsta.tech/get | sh
+        ```
+        or
+        ```
+        wget -qO- https://lepsta.tech/get | sh
+        ```
+
+    ![Get Uju](/assets/images/downloadUju.PNG)
```

Comment:
I don't think we need this especially since we didn't add it to the Linux and Mac side. Just for consistency, I think we should either remove it here or add screenshots dor Linux and Mac as well.

Reply:
Thank you for these suggestions. This part actually affects te Linux, Mac and windows part. Kindly review again and confirm.